### DISCUSSION TITLE
How can I make the tab background not transparent?

### DISCUSSION THREAD

**Discussion by mattfox27** [kinghat/tabbed-card#139] [General]
Using a custom background image but the tabs are transparent I want them to match the rest of the card so dark background
![tab card](https://github.com/kinghat/tabbed-card/assets/6801877/d4781a4e-acd1-4d1d-8d15-cc43e5178a0b)

**Comment by kinghat** [comment]
> Using a custom background image but the tabs are transparent I want them to match the rest of the card so dark background 

can you post a simplified config that has the same issue and the version youre using?
